### PR TITLE
publish stuff based off the right branch

### PR DESCRIPTION
**Pull request by krunal-patel-fl4re**

### CODE REVIEW

**Comment by krunal-patel-fl4re** [comment]
please accept pr #1 first

File: publish_binary_on_windows.bat [hunk 1]
```diff
@@ -0,0 +1,15 @@
+setlocal EnableDelayedExpansion
+
+git rev-parse --is-inside-work-tree
+if errorlevel 1 exit /b %errorlevel% 
+
+call npm --no-git-tag-version version patch
+if errorlevel 1 exit /b %errorlevel%
+
+call npm install @sb/prades
```
Comment (mbarnes-sb) [inline publish_binary_on_windows.bat]: @krunal-patel-fl4re status of using`yarn` for such steps? Also not relying on global `prades`? This is unstable.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "file": "{package_name}/{package_version}/c-blosc-{package_version}-{platform}-{arch}.tar.gz",
+    "host": "https://registry-node.starbreeze.com/-/releases",
+    "os": "win32",
+    "path": [
+      "artifacts",
+      "artifacts/**"
+    ]
+  },
+  "scripts": {
+    "install": "prades install"
```
Comment (mbarnes-sb) [inline package.json:17]: @krunal-patel-fl4re standardize on binary dependency for `prades`? Let's talk offline (public repo)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@sb/c-blosc",
+  "version": "1.6.50",
```
Comment (krunal-patel-fl4re) [inline package.json:3]: because base repo is at 1.6.1

**Review by mbarnes-sb — APPROVED** [review]

**Review by mbarnes-sb — APPROVED** [review]
looks good.

**Pull request merged**